recipes/tagine.md

```diff
@@ -0,0 +1,53 @@
+# Lubes' Vegetable Tagine
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-2hr-orange.svg)
+
+
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+
+![Serves Badge](https://img.shields.io/badge/Serves-12-green.svg)
+![Leftovers Badge](https://img.shields.io/badge/Leftovers-false-red.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Generally%20positive-green.svg)
+
+
+> A deliciously sweet and aromatic vegetable tagine, which subtly combines local and international tastes to achieve a vegan-friendly mid-week treat for the whole family. 
+
+
+| date first cooked | FACN2 week 4 day 3 |
+| ---| --- |
+| head chef | Lubes |
```

Comment:
don't think they're needed, might encourage clickthroughs and lead people away from other recipes.